src/django/contricleaner/lib/source_parser_csv.py

```diff
@@ -0,0 +1,14 @@
+
+from typing import List
+from contricleaner.lib.source_parser import SourceParser
+
+
+class SourceParserCsv(SourceParser):
+    def __init__(self):
+        pass
+
+    def parsed_header(self) -> List[str]:
+        pass
+
+    def parsed_rows(self) -> List[dict]:
+        pass
```

Comment:
Why it empty with implementation, is it planed?

Reply:
Oh I remember, .csv will be only for ListParser and now SourceParser, am I right? @killev

Reply:
Yes, you're right. I have implementation already